src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,60 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.db.Inventory;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.BalanceOperation;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.DataConverterImpl;
+import core.basesyntax.service.FileRead;
+import core.basesyntax.service.FileReadImpl;
+import core.basesyntax.service.FileWrite;
+import core.basesyntax.service.FileWriteImpl;
+import core.basesyntax.service.FruitOperationHandler;
+import core.basesyntax.service.PurchaseOperation;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ReportGeneratorImpl;
+import core.basesyntax.service.ReturnOperation;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.SupplyOperation;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class HelloWorld {
-    // HINT: In the `public static void main(String[] args)` it is better to create instances of your classes, 
-    // and call their methods, but do not write any business logic in the `main` method!
+    public static void main(String[] arg) {
```

Comment:
Don't begin method implementation with an empty line. Please remove the redundant empty line at the start of the `main` method.